src/insns/wavedrom/gctype.adoc

```diff
@@ -0,0 +1,12 @@
+
+[wavedrom, ,svg]
+....
+{reg: [
+  {bits: 7,  name: 'opcode',  attr: ['7', 'OP=0110011'], type: 8},
+  {bits: 5,  name: 'rd',      attr: ['5', 'dest'], type: 2},
+  {bits: 3,  name: 'funct3',  attr: ['3', 'GCTYPE=000'], type: 8},
+  {bits: 5,  name: 'cs1',     attr: ['5', 'src'], type: 4},
+  {bits: 5,  name: 'funct5',  attr: ['5', 'GCTYPE=00010'], type: 3},
```

Comment:
Looking at https://github.com/tariqkurd-repo/riscv-opcodes/blob/tariqkurd-repo/master/unratified/rv64_cheri, the getter with value 2 is not allocated right now, so I used that one.

the CSV files still need updating but I was going to wait until we have agreement on this PR.

Reply:
Updated CSV now.